norse/torch/functional/receptive_field.py

```diff
@@ -158,9 +168,21 @@ def spatial_receptive_fields_with_derivatives(
     ]
 
 
-def temporal_scale_distribution(start: float, stop: float, steps: int):
-    return torch.exp(
-        torch.linspace(
-            torch.log(torch.as_tensor(start)), torch.log(torch.as_tensor(stop)), steps
-        )
-    )
+def temporal_scale_distribution(
+    n_scales: int, min_scale: float = 1, c: float = 1.41421
+):
+    r"""
+    Provides temporal scales according to [Lindeberg2016].
+    The scales will be logarithmic by default, but can be changed by providing other values for c.
+    .. math:
+        \tau_k = c^{2(k - K)} \tau_{max}
+        \mu_k = \sqrt(\tau_k - \tau_{k - 1})
+    Arguments:
+      n_scales (int): Number of scales to generate
+      min_scale (float): The minimum scale
+      c (float): The base from which to generate scale values. Should be a value between 1 to 2
```

Comment:
Following the pattern from the previous sampling distributions (min_val, max_val, num_of_values), shouldn't the scales be selected from min and max values instead of the base c? [Lindberg 3.4: _Logarithmic Distribution of the Temporal Scales with Given Minimum Scale_](https://link.springer.com/article/10.1007/s10851-015-0613-9)

Reply:
I can see this making sense in two ways: either to have a min/max setting, as you describe, or to provide `min_scale` and `c` to automatically spread out the scales. Perhaps we should allow both?

Reply:
What do you think about the current solution?